src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -44,21 +48,37 @@ import { ImageBuilderHeader } from '../sharedComponents/ImageBuilderHeader';
 
 type CustomWizardFooterPropType = {
   disableNext: boolean;
+  isLoading?: boolean;
+  onSubmit?: () => void;
+  nextButtonText?: string;
 };
 
 export const CustomWizardFooter = ({
   disableNext: disableNext,
+  nextButtonText = 'Next',
+  isLoading,
+  onSubmit,
```

Comment:
I might be missing something, but are any of these changes to `<CustomWizardFooter>` used in this pull request? It kind of looks like it might be from an early iteration before you extracted everything into `<ReviewWizardFooter>` but maybe I'm not seeing something...

Reply:
And FWIW I think having a separate component `<ReviewWizardFooter>` is exactly what we should do, these footers are quite small and simple so it is better to just have separate components.

Reply:
Yes, it was a leftover from early iteration, I kept the ouid tho.